### PR TITLE
Add a dev setup with docker-compose

### PR DESCRIPTION
**Pull request by pior**
@mlhamel @nilovna

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    volumes:
+      - .:/app
+    ports:
+      - "8000:8000"
+    depends_on:
+      - db
+    environment:
+      - DEBUG=1
+      - SECRET_KEY=devsecret
+      - DATABASE_URL=postgres://postgres:mysecretpassword@db:5432/postgres
```
Comment (mlhamel) [inline docker-compose.yml]: 😢

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+FROM python:2.7
```
Comment (mlhamel) [inline Dockerfile]: we should try to switch to 3.5 or event 3.6 or pypy :)

File: docker-compose.yml (hunk 1 above)
Comment (pior) [inline docker-compose.yml]: Are you sarcastically crying? And also, is it a *good* thing?

Comment (mlhamel) [inline docker-compose.yml]: no there's just a missing new line :)

File: Dockerfile (hunk 2 above)
Comment (merwok) [inline Dockerfile]: Docker et Python 3?  Trop de futur en même temps man

**Pull request merged**